app/views/recipes/index.html.erb

```diff
@@ -0,0 +1,20 @@
+<div class='main_food_div'>
+  <h1>My Recipes</h1>
+  <%= link_to 'Add New Recipe', new_recipe_path, class: 'add_food' %>
+  <div class="recipes">
+  
+    <% @recipes.each do |recipe| %>
```

Comment:
- [Optional] Hi team, I think we can logically render a text: `No Recipe Added Yet ` when no recipe have been added for better users experience :+1:  instead of keeping this page empty without any info